src/main/java/ch/njol/skript/effects/EffBroadcast.java

```diff
@@ -42,49 +42,69 @@
 		"message to all players instead of broadcasting it."})
 @Examples({"broadcast \"Welcome %player% to the server!\"",
 		"broadcast \"Woah! It's a message!\""})
-@Since("1.0")
+@Since("1.0, INSERT VERSION (send multiple times)")
 public class EffBroadcast extends Effect {
+
 	static {
-		Skript.registerEffect(EffBroadcast.class, "broadcast %strings% [(to|in) %-worlds%]");
+		Skript.registerEffect(EffBroadcast.class, "broadcast %strings% [(to|in) %-worlds%] [%number% times]");
 	}
-	
+
 	@SuppressWarnings("null")
 	private Expression<String> messages;
+
 	@Nullable
 	private Expression<World> worlds;
-	
+
+	@Nullable
+	private Expression<Number> repeat;
+
 	@SuppressWarnings({"unchecked", "null"})
 	@Override
-	public boolean init(final Expression<?>[] vars, final int matchedPattern, final Kleenean isDelayed, final ParseResult parser) {
-		messages = (Expression<String>) vars[0];
-		worlds = (Expression<World>) vars[1];
+	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parser) {
+		messages = (Expression<String>) exprs[0];
+		worlds = (Expression<World>) exprs[1];
+		repeat = (Expression<Number>) exprs[2];
 		return true;
 	}
-	
+
 	@Override
 	public void execute(final Event e) {
+		int times = 1;
+		if (repeat != null) {
+            Number n = repeat.getSingle(e);
+            if (n != null) {
+            	times = n.intValue();
+            	if (times < 1) times = 1;

```

Comment:
Yeah, definitely think this should be clamped to 0 rather than 1. This just breaks expectations and takes control away from the user; if they're using a variable that can be < 1 to control the number of messages and they want it to always send at least 1, they can handle that themselves.